site/src/jsMain/resources/markdown/docs/guides/ExistingBackend.md

```diff
@@ -16,20 +16,19 @@ example, serving these files is a one-liner:
 
```

Comment:
The comment above is slightly incorrect now. How about:

When you export a site statically, it will generate all files into your `.kobweb/site` folder. Then, if using Ktor, for
example, you can serve these files using their [`staticFiles` method](https://ktor.io/docs/server-static-content.html):